### PR TITLE
extended: bump image tagging timeout

### PR DESCRIPTION
**Pull request by kargakis**
@mfojtik ptal

Fixes https://github.com/openshift/origin/issues/10663

### CODE REVIEW

**Comment by kargakis** [comment]
[test]

**Comment by mfojtik** [comment]
LGTM [merge]

File: test/extended/deployments/deployments.go [hunk 1]
```diff
@@ -263,7 +263,7 @@ var _ = g.Describe("deploymentconfigs", func() {
 
 			g.By("verifying the post deployment action happened: tag is set")
 			var out string
-			pollErr := wait.PollImmediate(100*time.Millisecond, 10*time.Second, func() (bool, error) {
+			pollErr := wait.PollImmediate(100*time.Millisecond, 1*time.Minute, func() (bool, error) {
```
Comment (soltysh) [inline test/extended/deployments/deployments.go]: Do we really, really want to go that high? This significantly extends our extended tests. Isn't something around 30s sufficient?

Comment (kargakis) [inline test/extended/deployments/deployments.go]: > This significantly extends our extended tests.

What do you mean by this? A bunch of extended tests use up to 5m timeouts.

Comment (soltysh) [inline test/extended/deployments/deployments.go]: I'm aware of 5 mins here and there, but the less we have them the better for us. Those timeouts cost us longer build/test times in the queue. On the other hand flakes cost us repeated times, as well :/

**Pull request merged**